lib/hammer_cli/output/adapter/table.rb

```diff
@@ -1,5 +1,6 @@
 require 'table_print'
 require File.join(File.dirname(__FILE__), 'wrapper_formatter')
+require File.join(File.dirname(__FILE__), '../../../table_print/formatter')
```

Comment:
doesn't just `'table_print/formatter'` work? if the gem is available I'd expect it to work